tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -699,8 +699,12 @@ def concatenate(self, dataset):
   def prefetch(self, buffer_size):
     """Creates a `Dataset` that prefetches elements from this dataset.
 
-    Note that if the dataset was batched using `Dataset.batch`, each element is
-    a batch and this operation will prefetch `buffer_size` batches.
+    Note: Like other `Dataset` methods, prefetch operates on the
+    elements of the dataset. It has no concept of examples vs. batches.
+    `examples.prefetch(2)` will prefetch two elements (2 examples),
+    while `examples.batch(20).prefetch(2)` will prefetch 2 elements
+    (2 batches, of 20 examples each). The typical use-case is to
```

Comment:
Remove the last sentence. It might a typical use case for yourself, but not for others.